libs/feature/search/src/lib/results-list/results-list.container.component.html

```diff
@@ -8,9 +8,15 @@
   ></gn-ui-results-list>
 
   <ng-container *ngIf="(facade.isLoading$ | async) === false">
-    <div class="show-more" *ngIf="showMore === 'button'">
-      <gn-ui-button type="primary" (buttonClick)="onShowMore()">
-        <span translate>results.showMore</span>
+    <div class="show-more mt-[30px]" *ngIf="showMore === 'button'">
+      <gn-ui-button
+        type="secondary"
+        (buttonClick)="onShowMore()"
+        extraClass="lg:m-auto !p-[22px]"
```

Comment:
Could it be an option to center the button on small screens, as well ? In particular, on the timeline, the button is currently "pressed" to the left border of the screen (without any margin).

Reply:
good point, I thought it would be better to have it on the left but it doesn't look that good actually, centered is fine!